sympy/tensor/arraypy.py

```diff
@@ -0,0 +1,1414 @@
+# -*- coding: utf-8 -*-
+
+from sympy import Symbol
+from sympy.matrices import Matrix, MatrixSymbol
+from copy import copy
+from sympy import sympify
```

Comment:
`from sympy.core.compatibility import string_types, range`
This makes sure that `range` behaves the same way in Python 2 and Python 3.

Furthermore, strings in Python 3 are slightly different. SymPy offers `string_types` to solve compatibility issues between Python 2 and 3.